PinpointKit/PinpointKit/Sources/AnnotationView.swift

```diff
@@ -8,18 +8,16 @@
 
 import UIKit
 
+/// The alpha value used for annotation borders.
+let BorderAlpha: CGFloat = 0.7
+
 /// The base annotation `UIView` subclass.
-public class AnnotationView: UIView {
-    
-    /// The alpha value used for annotation borders.
-    static let BorderAlpha: CGFloat = 0.7
+public protocol AnnotationView {
```

Comment:
Since this protocol no longer necessitates implementation on just `UIView` objects, but instead _has_ a view object, I'm thinking another name might better convey what it does. Maybe `AnnotationControlling, `AnnotationBehaviorControlling`, `AnnotationAppearanceControlling`, or something along those lines?